pkg/node/address.go

```diff
@@ -517,7 +525,7 @@ func chooseHostIPsToRestore(ipv6 bool, fromK8s, fromFS net.IP, cidrs []*cidr.CID
 	}
 
 	for _, cidr := range cidrs {
-		if cidr != nil && cidr.Contains(ip) {
+		if cidr != nil && cidr.String() != "" && cidr.Contains(ip) {
```

Comment:
Why is this change necessary? Shouldn't `cidr != nil` result in the same thing as `cidr.String() != ""`?